control/control_plane_core.go

```diff
@@ -275,7 +242,7 @@ func (c *controlPlaneCore) setupRoutingPolicy() (err error) {
 			}
 		}
 		if errs != nil {
-			return fmt.Errorf("IpRouteDel(lo): %w", errs)
+			c.log.Warnln("IpRouteDel: ", errs)
```

Comment:
Changed to `c.log.Debugf()`. In my opinion it may be unnecessary to delete ip routes at all,  because they are gone with netns, will be erased by kernel as long as netns is removed. (context: routes are inside netns now)

Reply:
Are duplicated rules possible when reloading?